integration/airflow/openlineage/airflow/facets.py

```diff
@@ -27,3 +28,21 @@ def from_task(cls, task):
 @attr.s
 class AirflowRunArgsRunFacet(BaseFacet):
     externalTrigger: bool = attr.ib(default=False)
+
+
+@attr.s
+class UnknownOperatorInstance:
+    """
+    Describes an unknown operator - specifies the (class) name of the operator and its properties
+    """
+    name: str = attr.ib()
+    properties: Dict[str, object] = attr.ib()
+    type: str = "operator"
+
+
+@attr.s
+class UnknownOperatorAttributeRunFacet(BaseFacet):
+    """
+    RunFacet that describes unknown operators in an Airflow DAG
+    """
+    unknownItems: List[UnknownOperatorInstance] = attr.ib()
```

Comment:
Do we need to be generic and use `unknownItems` or can we use `unknownOperatorClasses`?